managed-services/transactional-email/reference-content/tem-capabilities-and-limits.mdx

```diff
@@ -57,3 +57,14 @@ The attachments available are the following:
   - The recursive SPF `include` directive is not supported.
   - The subject, body and HTML of your emails must have at least 10 characters.
 </Message>
+
+## Supported ESMTP extensions
+
+Transactional Email's SMTP relay supports the following ESMTP extensions:
```

Comment:
Maybe add a link that explains what are these extensions? E.g. https://www.iana.org/assignments/mail-parameters/mail-parameters.txt

(maybe you can find a better one, this is the official IANA link but it is not very beautiful :))

Reply:
I'll use the IANA one for now! Thanks Bene!